nodejs/week2/meals/src/backend/index.js

```diff
@@ -0,0 +1,35 @@
+const express = require("express");
+const app = express();
+const port = process.env.PORT || 3000;
+
+// Routers - Inside the index.js file create routes using express
+app.get("/", (req, res) => res.send("Hello World"));
+
+ const meals = require("./routes/meals.js");
+ app.get("/meals", meals);
+
+
+//Respond with the json for the meal with the corresponding id
+
+const getMealWithId = require("./routes/meals.js");
+app.get("/meals/:id", getMealWithId);
```

Comment:
It is not necessity to have both in index.js
```
app.get("/meals", meals);
app.get("/meals/:id", meals);
```
You can use only one specific router fx
```
// in index.js
app.use('/meals', meals);
// in mealsRouter.js
router.get('/', (req, res) => { /* return meals */ }
router.get('/:id', (req, res) => { /* return a meal by id */ }
```

The same issue for reviews and reservations.